Src/Core/Collections/CollectionAssertions.cs

```diff
@@ -209,65 +207,33 @@ public AndConstraint<TAssertions> Equal(IEnumerable expected, string because = "
         protected void AssertSubjectEquality<TActual, TExpected>(IEnumerable expectation, Func<TActual, TExpected, bool> predicate,
             string because = "", params object[] reasonArgs)
         {
-            AssertionScope assertion = Execute.Assertion.BecauseOf(because, reasonArgs);
-
-            bool subjectIsNull = ReferenceEquals(Subject, null);
-            bool expectationIsNull = ReferenceEquals(expectation, null);
 
+            var subjectIsNull = ReferenceEquals(Subject, null);
```

Comment:
Sure, though "the type is very obvious from the same statement" is a bit vague. ReferenceEquals is a well-known method, though there is no real difference in maintainability between 'bool' and 'var', so it can what is easier to read. Another example - `ToArray()` produces an array, but type is unknown, though you can see the type because it is `Cast<TExpected>().ToArray()'.
Do you want me to replace all the `var` usages in the changeset?